docs/ops/deployment/kubernetes.md

```diff
@@ -166,18 +242,31 @@ kind: Service
 metadata:
   name: flink-jobmanager
 spec:
+  type: ClusterIP
   ports:
   - name: rpc
     port: 6123
   - name: blob
     port: 6124
-  - name: query
-    port: 6125
   - name: ui
     port: 8081
   selector:
     app: flink
     component: jobmanager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: flink-jobmanager-svc
```

Comment:
dumb question, does `svc` stand for service? If yes, then we could maybe rename it into `flink-web-ui-service`.

Reply:
Actually, this service enables user could submit jobs via shell script to rest API, how about `flink-rest-service`?